src/main/java/com/ecore/roles/service/impl/RolesServiceImpl.java

```diff
@@ -3,59 +3,40 @@
 import com.ecore.roles.exception.ResourceExistsException;
 import com.ecore.roles.exception.ResourceNotFoundException;
 import com.ecore.roles.model.Role;
-import com.ecore.roles.repository.MembershipRepository;
 import com.ecore.roles.repository.RoleRepository;
-import com.ecore.roles.service.MembershipsService;
 import com.ecore.roles.service.RolesService;
 import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.log4j.Log4j2;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.UUID;
 
 @Log4j2
 @Service
+@RequiredArgsConstructor
 public class RolesServiceImpl implements RolesService {
 
-    public static final String DEFAULT_ROLE = "Developer";
-
     private final RoleRepository roleRepository;
-    private final MembershipRepository membershipRepository;
-    private final MembershipsService membershipsService;
-
-    @Autowired
-    public RolesServiceImpl(
```

Comment:
This constructor can be replaced with @RequiredArgsConstructor lombok annotation in order to become the code cleaner